dimod/reference/samplers/exact_solver.py

```diff
@@ -159,6 +159,53 @@ def sample_poly(self, polynomial, **kwargs):
         """
         return ExactSolver().sample(polynomial, **kwargs)
 
+class ExactDQMSolver(Sampler):
+    """A simple exact solver for testing and debugging code using your local CPU.
+
+    Notes:
+        This solver becomes slow for problems with 18 or more
+        variables, worse for increasingly many cases per variable.
+
+    """
+    properties = None
+    parameters = None
+
+    def __init__(self):
+        self.properties = {}
+        self.parameters = {}
+
+    def sample_ising(self, h, J, **parameters):
+        return ExactSolver().sample_ising(h, J, **parameters)
+
+    def sample_qubo(self, Q, **parameters):
+        return ExactSolver().sample_qubo(Q, **parameters)
+
+    def sample(self, dqm, **parameters):
+        return self.sample_dqm(dqm, **parameters)
```

Comment:
If all the class does is sample DQM's, then I like the shortened version (as opposed to or in addition to `sample_dqm`), but it wouldn't be consistent with `LeapHybridDQMSampler`.  On the other hand, if you keep `sample_ising` and `sample_qubo`, `sample` might be ambiguous in this context.